Sources/App/Features/Push/Controllers/PushController.swift

```diff
@@ -11,7 +11,17 @@ struct PushController: RouteCollection {
 
     private func create(req: Request) async throws -> Token {
         let token = try req.content.decode(Token.self)
-        try await token.save(on: req.db)
+
+        // Either store or update the token value
+        if let storedToken = try await Token.query(on: req.db)
+            .filter(\.$token == token.token)
+            .first() {
+            storedToken.debug = token.debug
```

Comment:
What's the 'debug' here?

Reply:
It's to separate out debug keys, so hopefully will show which tokens to ignore if you're pointing to the service from a test build, as they'll never work from Production Push.